exercises/robot-simulator/src/test/java/RobotTest.java

```diff
@@ -0,0 +1,275 @@
+import org.junit.Ignore;
+import org.junit.Test;
+
+import static org.junit.Assert.assertTrue;
+
+public final class RobotTest {
+
+    @Test
+    public void testRobotIsCreatedWithCorrectInitialPosition() {
+        final GridPosition initialGridPosition = new GridPosition(0, 0);
+        final Robot robot = new Robot(initialGridPosition, Orientation.NORTH);
+
+        final GridPosition actualGridPosition = robot.getGridPosition();
+
+        assertTrue(actualGridPosition.equals(initialGridPosition));
+    }
+
+    @Ignore
+    @Test
+    public void testRobotIsCreatedWithCorrectInitialOrientation() {
+        final Orientation initialOrientation = Orientation.NORTH;
+        final Robot robot = new Robot(new GridPosition(0, 0), initialOrientation);
+
+        final Orientation actualOrientation = robot.getOrientation();
+
+        assertTrue(actualOrientation.equals(initialOrientation));
```

Comment:
I'm wondering if `actualOrientation` is carrying its intention-revealing weight, here.  How does this compare with:

```java
@Test
public void testRobotIsCreatedWithCorrectInitialOrientation() {
    final Orientation initialOrientation = Orientation.NORTH;
    final Robot robot = new Robot(new GridPosition(0, 0), initialOrientation);

    assertEquals(robot.getOrientation(), initialOrientation);
}
```

?